### ISSUE TITLE
Ad a filter to *ignore* a label

### ISSUE THREAD

**Issue by meder-yembo** [integrations/slack#1331]
**Is your feature request related to a problem? Please describe.**
I set up GitHub slack integration to help us better track real action items around a PR because things often get drowned out by bot and dependency PRs. However, I can't filter out the bot PRs, so this is equally as spammy.

**Describe the solution you'd like**
Add a filter that allows a user to ignore notifications tagged with a certain label. For example, ignore notifications about PRs tagged with the `bot` label.

**Describe alternatives you've considered**
GitHub actions to add a `not-a-bot` label automatically to PRs. This is required on every repository workflow though, so it's not as ideal as simply filtering all notifications at a single source.

**Additional context**
None.

**Comment by dominic-blain** [comment]
Would be so useful to ignore dependabot PRs

**Comment by k-sav** [comment]
Would like to ignore by text.  
`codeclimate[bot] commented on`

**Comment by neil-morrison44** [comment]
Yeah, we're holding off on using the slack integration in anything other than a test channel until we're able to do `-label:dependencies`.

I assume from the existing syntax being `+label:` the `-label:` equivalent is planned?

**Comment by luislew** [comment]
+1 - we label our Renovate PRs but would love to filter them out of Slack notifications

**Comment by naor-capitola** [comment]
If you just need PR notifications then GitHub's Scheduled Notification for your [organization](https://docs.github.com/en/organizations/managing-organization-settings/managing-scheduled-reminders-for-your-organization), [team](https://docs.github.com/en/organizations/organizing-members-into-teams/managing-scheduled-reminders-for-your-team) or [users](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-membership-in-organizations/managing-your-scheduled-reminders) worked for me. Allows you to filter in or out by labels and much more.
